python/internetarchive/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                internetarchive
+version             4.0.1
+revision            0
+
+categories-append   net
+license             AGPL-3
+maintainers         nomaintainer
+
```

Comment:
also here it looks like there are no architecture dependent files so please add the same lines as I mentioned before. I am not repeating comments in every port but assume that you'll check these things yourself for the others.